src/Frontend/menu/model/MenuItem.js

```diff
@@ -0,0 +1,24 @@
+/*
+ * This file is part of the sententiaregum application.
+ *
+ * Sententiaregum is a social network based on Symfony2 and ReactJS
+ *
+ * @copyright (c) 2015 Sententiaregum
+ * Please check out the license file in the document root of this application
+ */
+
+/**
+ * @author Benjamin Bieler <benjaminbieler2014@gmail.com>
+ */
+class MenuConfig {
```

Comment:
this should be moved to a global model directory.
*Frontend/model/menu/MenuConfig.js*